fast/stages/0-bootstrap/data/org-iam.tfvars.json

```diff
@@ -0,0 +1,112 @@
+{
+  "version": "0.1.0",
+  "organization": {
+    "id": "366118655033",
+    "name": "ludo.joonix.net"
+  },
+  "iam_bindings": [
+    {
+      "principal": "group:gcp-organization-admins@ludo.joonix.net",
+      "group_id": "ORG_ADMINS",
+      "role": [
+        "roles/resourcemanager.organizationAdmin",
+        "roles/resourcemanager.folderAdmin",
+        "roles/resourcemanager.projectCreator",
+        "roles/billing.user",
+        "roles/iam.organizationRoleAdmin",
+        "roles/orgpolicy.policyAdmin",
+        "roles/securitycenter.admin",
+        "roles/cloudsupport.admin",
+        "roles/owner",
+        "roles/cloudasset.owner",
+        "roles/compute.osAdminLogin",
+        "roles/compute.osLoginExternalUser",
+        "roles/resourcemanager.tagAdmin",
+        "roles/compute.xpnAdmin"
+      ],
+      "resource": {
+        "type": "ORGANIZATION",
+        "id": "656131167402"
```

Comment:
I noticed that this org id is not matching the org id on the top. But it is the top id that we verify that is in agreement (and then - I think it is where we do apply the IAM). Is this mistake in data or in logic?

Reply:
this is my org id for tests, once we move these files to test assets the org id will change :)